composer.json

```diff
@@ -114,7 +114,8 @@
     },
     "extra": {
         "symfony": {
-            "allow-contrib": true
+            "allow-contrib": true,
+            "require": "5.4.*"
```

Comment:
5.4 is the highest version supported by Sulu 2.4, the 2.5 will be currently 6.2 until we solve the 6.3 issues.